packages/vscode/package.json

```diff
@@ -145,6 +145,12 @@
             "/path/to/your/schema.prisma"
           ],
           "description": "If you have a Prisma schema file in a custom path, you will need to provide said path `/path/to/your/schema.prisma` to run generate"
+        },
+        "prisma.pdpNotification": {
+          "title": "Pdp Notification",
+          "type": "boolean",
+          "default": "true",
+          "description": "..."
```

Comment:
This is the definition for this setting
![image](https://github.com/prisma/language-tools/assets/29753584/dba96476-3b31-4de7-b456-4470149bac2d)

Reply:
This needs to be updated before merging because as a user it's very weird and impossible to understand what it does.
<img width="475" alt="Screenshot 2024-05-08 at 10 57 00" src="https://github.com/prisma/language-tools/assets/1328733/ae99dd01-f422-4b02-91b9-83ed1c45bf25">